### PR TITLE
alert_badge

### PR DESCRIPTION
**Pull request by Shailseen**
update alert_badge

### CODE REVIEW

File: component css/alert.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    color: orange;
+}
+.primary{
+    color: blueviolet;
+}
+.main-alert-section > .component-intro li{
+    list-style: none;
+}
+.alert-pack{
+    display: flex;
```
Comment (anupkgurung) [inline component css/alert.css:17]: what is alert pack??

File: component css/badge.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    background-color: var(--primary-color);
+}
+.badge-shopping-cart{
+    position: absolute;
+    margin-bottom: 2rem;
+    margin-left: 2rem;
+}
+.badge-outputs{
+    display: flex;
+    justify-content: center;
```
Comment (anupkgurung) [inline component css/badge.css]: .badge-avatar-container also used flex and center. If you are using common properties try to add all those under one class and use that instead.

File: component css/badge.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+}
+.fa-shopping-cart{
+    font-size: 2rem;
+    color: #047bd5;
+}
+.fa-paper-plane{
+    font-size: 3rem;
+    color: white;
+    background-color: #229ED9;
+    border-radius: 50%;
```
Comment (anupkgurung) [inline component css/badge.css:66]: hex code can be moved to varaible.

**Review by anupkgurung — APPROVED** [review]
You can refactor your code to analyze properties which are common and put them all into one separate file. And you can use that all over the application.

**Review by vishalg8454 — APPROVED** [review]
All looks fine. Ready to merge.

**Pull request merged**